capellambse/model/common/element.py

```diff
@@ -216,7 +218,7 @@ def __init__(
                         f"Cannot set {key!r} on {type(self).__name__}"
                     )
                 else:
-                    setattr(self, key, val)
+                    self.__setattr__(key, val, audit=audit)
```

Comment:
The only change that I don't like so much. But I have no idea how to do it such that we can keep `setattr()`.